docs/applications/bitcoin-core.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: default
+title: Bitcoin Core
+parent: Main Applications
+nav_order: 610
+---
+## Bitcoin Core
+
+Bitcoin Core the most popular software implementation of Bitcoin, openly developed on GitHub (<https://github.com/bitcoin/bitcoin>) and publishing releases on <https://bitcoincore.org>. The BitBox base uses the latest stable binary release to communicate with the Bitcoin peer-to-peer network, for example to learn about current transactions and receive newly mined blocks. All data is regarded as untrusted until it is locally verified to the consensus rules enforced by the BitBox Base.
```

Comment:
I feel that saying that `the consensus rules enforced by the BitBox Base` in the context of describing Bitcoin Core is a little odd.. why not say that `the consensus rules verified by Bitcoin Core`?

(I.e `Bitcoin Core`, not `BitBox Base`, as it is the component that verifies the rules, and `verified`, not `enforced`, as there's no real way to force the rest of the Bitcoin network to do anything by running a certain program, but we can verify that the blocks and transactions we receive are following the correct rules.)

Reply:
changed to `...until it is locally verified to match the consensus rules of Bitcoin Core`